ragaai_catalyst/tracers/exporters/ragaai_trace_exporter.py

```
@@ -206,13 +207,21 @@ def upload_trace(self, ragaai_trace_details, trace_id):
 
         logger.info(f"Submitted upload task with ID: {self.upload_task_id}")
     
-    async def upload_rag_trace(self, ragaai_trace, additional_metadata, trace_id):
+    async def upload_rag_trace(self, ragaai_trace, additional_metadata, trace_id, post_processor=None):
         try:
             ragaai_trace[0]['external_id'] = self.external_id
             trace_file_path = os.path.join(self.tmp_dir, f"{trace_id}.json")
             with open(trace_file_path, 'w') as f:
                 json.dump(ragaai_trace, f, indent=2)
             logger.info(f"Trace file saved at {trace_file_path}")
+            if self.post_processor!=None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)